### PR TITLE
Invalid MA is ignored

### PR DESCRIPTION
**Pull request by diegomrsantos**

### CODE REVIEW

File: libp2p/multiaddress.nim [hunk 1]
```diff
@@ -1128,6 +1131,5 @@ proc getRepeatedField*(pb: ProtoBuffer, field: int,
       if ma.isOk():
         value.add(ma.get())
       else:
-        value.setLen(0)
-        return err(ProtoError.IncorrectBlob)
+        debug "Not supported MultiAddress in blob", ma = item
```
Comment (Menduist) [inline libp2p/multiaddress.nim:1134]: This doesn't seems like the best solution
Maybe create a `tryGetRepeatedField`, that returns the number of failed parses? This way the caller can handle this however it wants

Comment (Menduist) [reply libp2p/multiaddress.nim:1134]: If the current version is not relevant anywhere, then we should change this proc to return the number of failed parses (or eq)

Comment (Menduist) [reply libp2p/multiaddress.nim:1134]: And update every usage accordingly (and check it's not used by nimbus & co)

Comment (diegomrsantos) [reply libp2p/multiaddress.nim:1138]: pushed changes

**Review by Menduist — APPROVED** [review]

**Pull request merged**